### PR TITLE
🔖(chore) bump release to version 2.12.0

### PR DESCRIPTION
**Pull request by sampaccoud**
### Added

- Add catalog visibility field to course run so it's possible to hide a course run on the catalog and/or hide it completely. Possible values are `course_and_search`, `course_only` and `hidden`.

### Changed

- Do not use absolute positioning to place course glimpse metadata
- Improve semantic of glimpses

### Fixed

- Fixed a layout issue on course glimpse when organization title and course title are too long
- Add missing language parameter in LTI requests issued by LTI consumer plugin

### CODE REVIEW

File: website/versions.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [
-  "2.11.0",
+  "2.12.0",
+  "2.12.0",
```
Comment (jbpenrath) [inline website/versions.json:3]: Hmm, something went wrong doc has been upgraded.

```suggestion
  "2.12.0",
  "2.11.0",
```

Comment (sampaccoud) [reply website/versions.json:3]: WTF. Ah. ok.

**Review by jbpenrath — APPROVED** [review]
Leeeets goooo !

**Comment by igobranco** [comment]
Sorry @sampaccoud  didn't had time yesterday to review.

**Pull request merged**